src/main/java/lotto/Validate.java

```diff
@@ -0,0 +1,47 @@
+package lotto;
+
+import java.util.List;
+
+public class Validate {
```

Comment:
클래스 이름은 동사보다는 명사로 해줘야 해요!
```suggestion
public class Validator {
```